packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/profile.astro

```diff
@@ -41,46 +41,48 @@ const pageTitle = `User profile | ${contextConfig.title}`;
     description={contextConfig.description} >
 
     <div class="flex grow w-full pb-4">
-        <sl-breadcrumb>
-            <span slot="separator">/</span>
-            <sl-breadcrumb-item href={dashboardIndex}>Dashboard
-                <sl-icon slot="prefix" name="columns-gap"></sl-icon>
-            </sl-breadcrumb-item>
-            <sl-breadcrumb-item href={userProfile}>User Profile</sl-breadcrumb-item>
-        </sl-breadcrumb>
+        <Breadcrumb>
+            <BreadcrumbItem href={dashboardIndex} prefix='columns-gap'>
+                Dashboard
+            </BreadcrumbItem>
+            <BreadcrumbItem href={userProfile}>
+                User Profile
+            </BreadcrumbItem>
+        </Breadcrumb>
     </div>
 
     <div class="container">
 
         <h1 class="text-5xl font-bold py-8">User Profile</h1>
 
 
-        <sl-card class="card-header">
+        <Card class="card-header">
             <div slot="header" class="text-5xl font-bold">
                 {username}'s Profile
             </div>
+            <Divider class="divider" />
             <div class="py-2">
                 <text class="text-lg">
                     <bold>Display Name:</bold> {name}
```

Comment:
`bold` is now `strong`

Reply:
Unocss accepts and converts this... it could also be `<text-bold>` if i enable the attributify addon